packages/js/src/integrations-page/simple-integration.js

```diff
@@ -105,6 +105,8 @@ SimpleIntegration.propTypes = {
 		name: PropTypes.string,
 		claim: PropTypes.string,
 		learnMoreLink: PropTypes.string,
+		linkTextUnActive: PropTypes.string,
+		linkUnActive: PropTypes.string,
```

Comment:
I'm not sure about adding API like this, it is inherently dangerous to do if you do not adapt all the integrations themselves.
I guess I'd prefer either:
* adjusting all the integrations to implement these new props
* some clearer functionality surrounding how the fallback works

Other than that, it is `inActive`, not `unActive`.

Reply:
👍